helpers/gestures.js

```diff
@@ -0,0 +1,109 @@
+const maxTime = 2000;
+const minDistance = 30;
+
+export function registerSwipe(node) {
+	node.addEventListener('touchstart', handleTouchStart);
+}
+
+function handleTouchStart(e) {
+
+	const node = this; /* eslint-disable-line no-invalid-this */
+
+	let tracking = {
+		start: {
+			time: new Date().getTime(),
```

Comment:
Might want to use the [performance.now()](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) high resolution time stuff? It'll give you a much more accurate timestamp.